src/community/blocks.tsx

```diff
@@ -118,7 +118,7 @@ export const InformationBlock: React.FC<InformationBlockProps> = ({
         <Inline space="between" alignItems="flex-end" aria-label={ariaLabel}>
             <BlockContent title={title} description={description} />
             <div className={classNames(styles.column, styles.rightContent)}>
-                <Text2 regular color={vars.colors.textSecondary}>
+                <Text2 regular color={vars.colors.textSecondary} decoration="line-through">
```

Comment:
Can you add Figma specs in the PR Body? I think this decoration has to be configurable

Reply:
@yceballost This field is supposed to always use the line-through decoration, it's made to show the original price of the item.